blueprints/ember-decorators/index.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
+  normalizeEntityName: function() {}, // no-op since we're just adding dependencies
+
+  afterInstall: function() {
+    return this.addPackagesToProject([
+      { name: 'babel-plugin-transform-class-properties' },
```

Comment:
I actually tried doing this at first and it wouldn't build, and I was hesitant about taking control from consumers. Will try again, I'm sure my config was just messed up somehow